### ISSUE TITLE
what's this?

### ISSUE THREAD

**Issue by WU-SUNFLOWER** [Miracle-Eagle/PvZ2-Code-Port#1]
Do you mean you want to re-develop pvz2 program?

**Comment by Miracle-Eagle** [comment]
No, I just want to try to rewrite what would be its source code and made it crossplatform (kinda hars but will try)
